### PR TITLE
chore: test image for stat tags with new base image

### PR DESCRIPTION
**Pull request by sanpj2292**
## Description of the change

> Description here

## Type of change
- [ ] Bug fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Related issues

> Fix [#1]() 

### Development

- [ ] Lint rules pass locally
- [ ] The code changed/added as part of this pull request has been covered with tests
- [ ] All tests related to the changed code pass in development

### Code review 

- [ ]  This pull request has a descriptive title and information useful to a reviewer. There may be a screenshot or screencast attached
- [ ] "Ready for review" label attached to the PR and reviewers mentioned in a comment
- [ ] Changes have been reviewed by at least one other engineer
- [ ] Issue from task tracker has a link to this pull request

### CODE REVIEW

**Review by koladilip — DISMISSED** [review]
looks good except, \"\" in error messages is odd, do we require it?

**Comment by koladilip** [comment]
Is this still needed? Jayc is doing right?

**Comment by sanpj2292** [comment]
Basically till @Jayachand completes the discussed thing about service separation we would want seploy this image into hosted. Otherwise again segmentation fault issue can re-appear. Hence we would need this PR to be alive till that point of time. Let me know if more clarification is required

File: Dockerfile [hunk 1]
```diff
@@ -1,19 +1,18 @@
-FROM node:14.20-alpine3.15
+FROM ubuntu:20.04
```
Comment (saikumarrs) [inline Dockerfile:1]: The changes in this file are not needed anymore. Let's revert before merging the PR.

**Comment by sanpj2292** [comment]
We can discard the changes in this branch. Hence closing this PR

**Pull request closed without merging**